src/redux/actions.tsx

```diff
@@ -0,0 +1,23 @@
+import { ADD_TODO, TOGGLE_TODO, REMOVE_TODO } from './types';
+import {IAddTodo, IRemoveTodo, IToggleTodo} from '../core/interfaces';
+
+export const addTodo = (text: string): IAddTodo => {
+  return {
+    type: ADD_TODO,
+    text
+  };
+};
+
+export const removeTodo = (id: string | number): IRemoveTodo => {
```

Comment:
why your id can be number or string?

Reply:
@AlexButov 
For now it's Date.now(), but it also may be id generated from mongodb which is string. So I'm not sure should it be number or string